maintainers/maintainer-list.nix

```diff
@@ -4785,6 +4785,12 @@
     githubId = 1447245;
     name = "Robin Gloster";
   };
+  gnd = {
```

Comment:
Please consider using your Github username as a maintainer tag. IMHO this makes it easier to communicate on GitHub.